travis-ci/managers/fuzzit.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+set -x
+set -u
+
+REPO_ROOT=${REPO_ROOT:-$(pwd)}
+
+sudo bash -c "echo 'deb-src http://archive.ubuntu.com/ubuntu/ xenial main restricted universe multiverse' >>/etc/apt/sources.list"
+sudo apt-get update -y
+sudo apt-get build-dep systemd -y
+sudo apt-get install -y ninja-build python3-pip python3-setuptools
+pip3 install meson
+
+cd $REPO_ROOT
+export PATH="$HOME/.local/bin/:$PATH"
+export SANITIZER=address,undefined
+tools/oss-fuzz.sh
+
+export FUZZING_TYPE=${1:-sanity}
+# Because we want Fuzzit to run on every pull-request and Travis/Azure doesnt support encrypted keys
+# on pull-request we use a write-only key which is ok for now. maybe there will be a better solution in the future
+export FUZZIT_API_KEY=7c1bd82fe0927ffe1b4bf1e2e86cc812b28dfe08a7080a7bf498e98715884a163402ee37ba95d4b1637247deffcea43e
+export FUZZIT_ADDITIONAL_FILES="./out/src/shared/libsystemd-shared-242.so"
+export FUZZIT_ARGS="--type ${FUZZING_TYPE} --branch "PR-${TRAVIS_PULL_REQUEST}-${TRAVIS_BRANCH}" --revision ${TRAVIS_COMMIT}"
```

Comment:
I'm not sure `${TRAVIS_BRANCH}` is helpful here when PRs are opened. In our case it's almost always "master". Maybe, it would make sense to use something like `PR-${TRAVIS_PULL_REQUEST}` in PRs and fall back to `${TRAVIS_BRANCH}` if `${TRAVIS_PULL_REQUEST}` is "false" (according to https://docs.travis-ci.com/user/environment-variables, it's "the pull request number if the current job is a pull request, “false” if it’s not a pull request"

Reply:
done.